contracts/StablecoinConverter.sol

```diff
@@ -11,6 +11,8 @@ contract StablecoinConverter is EpochTokenLocker {
     using BytesLib for bytes32;
     using BytesLib for bytes;
 
+    uint constant MAX_UINT128 = 340282366920938463463374607431768211455;
```

Comment:
please 2^128 - 1

Reply:
Yes, this should work in solidity

```
uint x = 2**128 - 1;
```

or, if necessary,

```
uint128 x = uint128(2**128 - 1);
```